app/assets/javascripts/admin/addon/templates/backups-index.hbs

```diff
@@ -1,14 +1,18 @@
 <div class="backup-options">
   {{#if localBackupStorage}}
-    {{resumable-upload
-      target="/admin/backups/upload"
-      success=(route-action "uploadSuccess")
-      error=(route-action "uploadError")
-      uploadText=uploadLabel
-      title="admin.backups.upload.title"
-      class="btn-default"}}
+    {{#if siteSettings.enable_experimental_backup_uploader}}
+      {{uppy-backup-uploader done=(route-action "uploadSuccess") localBackupStorage=localBackupStorage}}
+    {{else}}
+      {{resumable-upload
+        target="/admin/backups/upload"
+        success=(route-action "uploadSuccess")
+        error=(route-action "uploadError")
+        uploadText=uploadLabel
+        title="admin.backups.upload.title"
+        class="btn-default"}}
+    {{/if}}
   {{else}}
-    {{#if enableExperimentalBackupUploader}}
+    {{#if (and siteSettings.enable_direct_s3_uploads siteSettings.enable_experimental_backup_uploader)}}
```

Comment:
So I have to turn on both site settings to use uppy? What happens when I have only `enabled_direct_s3_uploads` or only `enabled_experimental_backup_uploader`? From slightly higher above, it feels like experimental backup uploader is uppy's uploader and is direct uploads. Just wondering if it's needed to have both.

Reply:
If it's not local, and direct s3, and experimental, use uppy
Otherwise, use backup-uploader.

What happens if an admin sets the `enable_direct_s3_uploads` flag but not the experimental one? Does backup-uploader already support direct s3 uploads?

Mostly my lack of context asking.

Reply:
`enable_direct_s3_uploads` is also used for other things and will stay even when the `enable_experimental_backup_uploader` setting is removed. It's a catch-all because you need to have special permissions and other things set up in S3 to use direct S3 uploads. I also have a TODO in uppy-backup-uploader to address some things to allow for direct S3 upload _without_ multipart. So for now you need to have both these settings enabled to get the uppy backup uploader when going remote instead of local.